build_for_container

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "Building for $BUILDARCH"
+
 set -euo pipefail
```

Comment:
We usually initialize error handling first thing in the script, so let's move this above the `echo`.

Also, an environment variable is a little non-obvious, especially since `set -u` makes its existence required; let's just use a command-line argument instead (and print a usage message if not present).  That should be okay UX since this is only invoked for debugging or by the Dockerfile.